scripts/lint.js

```diff
@@ -31,29 +30,25 @@ const watcher = (label, pckgName, type = 'front') => {
   shell.echo('');
 };
 
-const files = glob
-  .sync('**/*.js', { ignore: '**/node_modules/**' })
-  .filter(f => changedFiles.has(f))
-  .filter(
-    package =>
-      !package.includes('README.md') &&
-      !package.includes('strapi-middleware-views') &&
-      !package.includes('strapi-lint') &&
-      !package.includes('strapi-plugin-settings-manager') &&
-      !package.includes('scripts') &&
-      !package.includes('test') &&
-      !package.includes('jest.config.js')
-  )
+const except = [
+  'jest.config.js',
+  'scripts',
+  'strapi-lint',
+  'strapi-middleware-views',
+  'strapi-plugin-settings-manager',
+  'test',
+];
```

Comment:
Why are we excluding this? 🤔

Reply:
Because when we added the lint the script was slightly different and those files did not pass the pre-commit hook... And I did have the time to adapt the code to respect the lint config

Reply:
Understood! Thanks for your response 👍